tasks/promote.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Ensure old images are retired
+  command: >
+            {{ os_images_venv }}/bin/openstack image set {{ promotion_name }} --name {{ promotion_name }}.{{ date_suffix }}
```

Comment:
either you do a multiline, or drop > ;-)

Reply:
Done